axum-extra/Cargo.toml

```diff
@@ -80,14 +80,14 @@ rustdoc-args = ["--cfg", "docsrs"]
 allowed = [
     "axum",
     "axum_core",
+    "axum_macros",
     "bytes",
     "cookie",
     "futures_core",
     "futures_util",
     "http",
     "http_body",
     "hyper",
-    "percent_encoding",
```

Comment:
Not sure why `percent_encoding` was here before. It's `#[doc(hidden)]` so shouldn't show up in the docs which `cargo-public-api-crates` generates. Might have been left over from a previous bug 🤷

Reply:
`cargo-public-api-crates` probably should detect crates in this list which _aren't_ in the public API.